src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -112,19 +112,34 @@ def two_groups_hyphens_two_replacements_with_suffix(
     "beautifulsoup4": ("bs4",),
     "bitvector": ("BitVector",),
     "cattrs": ("cattr", "cattrs"),
+    "django-auditlog": ("auditlog",),
```

Comment:
It looks like a lot (but not all) of these `django-...` ones might be covered by the `django-...` catch all rule above: https://github.com/pantsbuild/pants/blob/73d41b4c9935d18a2aa94c3bff6d8f99e0cc0dcb/src/python/pants/backend/python/dependency_inference/default_module_mapping.py#L88

In particular, the ones that map from `django-foo-bar` to `foo_bar` should be automatic. Could you adjust to only include the ones like `django-oauth-toolkit` that don't have the simple mapping?

(NB. potentially you may observe different behaviour locally: that catch all was only added in Pants 2.19.)

Reply:
I wasn't aware of the catch-all (still using pants v2.17) - thanks!

Reply:
@huonw Note I've added two non-standard django cases (`notifications` and `actstream`) after removing the packages covered by the catch-all